LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 PowerShell 6.0
 
-Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If we follow the copyright text in other files, we should keep this one here and remove line 5.